components/inputgroup/skin.css

```diff
@@ -118,7 +118,7 @@ governing permissions and limitations under the License.
 .spectrum-Datepicker--range {
   .spectrum-InputGroup-field {
     &:focus-ring {
-      box-shadow: none !important;
+      box-shadow: none;
```

Comment:
Still some work with `!important` left in this pr.

Reply:
Took it off but as I mentioned on the other PR, the best way to address this would be solving the css conflicts without the use of !important as it doesn´t complies with css standards. I don´t have the time for that right now :(